lib/tdf3/src/models/manifest.ts

```diff
@@ -6,4 +6,5 @@ export type Manifest = {
   payload: Payload;
   encryptionInformation: EncryptionInformation;
   assertions: Assertion[];
+  tdf_spec_version: string;
```

Comment:
it looks like this is importing [this file](https://github.com/opentdf/web-sdk/blob/main/lib/tdf3/src/version.ts) which specifies version `0.2.0` of the TDF spec. Is that what we should be doing in java?

Reply:
No, this should somehow be the spec version, which AFAIK is 4.3.0

https://github.com/opentdf/spec/blob/3c5adf29fe9c327b2f85b89e8beee09f0b91d8c7/VERSION